packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/core",
-  "version": "2.13.0",
+  "version": "2.13.1-alpha.1",
```

Comment:
Bump vue and react packages along with this

Reply:
Done!